codecov.yml

```diff
@@ -0,0 +1,22 @@
+coverage:
+  precision: 2        
+  round: down         
+  range: 20...80     
+  status:
+    project:                   
+      default:                 
+        enabled: yes           
+        target: 80%                 
+        if_no_uploads: error   
+        if_not_found: error  
```

Comment:
option  should be one of 'success' and 'failure', refer to https://docs.codecov.com/docs/commit-status#if_not_found

Reply:
Thanks for the comment, it has been updated.